firmware/thruna/PDM/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ file(GLOB_RECURSE APP_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/src/app/*.c")
 list(APPEND APP_SRCS 
     "${SHARED_APP_INCLUDE_DIR}/App_SharedStateMachine.c"
     "${SHARED_APP_INCLUDE_DIR}/App_SharedHeartbeatMonitor.c"
+    "${SHARED_APP_INCLUDE_DIR}/App_Timer.c"
```

Comment:
Why was this added?

Reply:
I thought I needed this for testing, for the LetTimePass() function???